docs/content/references/sui-api/graphql/guides.mdx

```diff
@@ -0,0 +1,257 @@
+---
+title: Guides
+description: Guidelines for working with the Sui GraphQL RPC.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+For an excellent introduction to GraphQL, please see [GitHub's Introduction to GraphQL page](https://docs.github.com/en/graphql/guides/introduction-to-graphql).
+
+## Discovering the schema
+GraphQL offers an introspection-like service that allows you to query for the schema that is loaded by a service instance. The [official documentation](https://graphql.org/learn/introspection/) has an excellent overview on introspection.
+
+## Querying the service
+
+### cURL like querying
+
+The endpoint can also be queried using using REST calls by passing the query to the call:
+```shell
+# basic query
+curl -X POST https://graphql-beta.mainnet.sui.io \
+     --header "Content-Type: application/json" \
+     --data '{
+          "query": "query { epoch { referenceGasPrice } }"
+     }'
+# query with variables
+curl -X POST https://graphql-beta.mainnet.sui.io \
+     --header "Content-Type: application/json" \
+     --data '{
+  "query": "query ($epochID: Int!) { epoch(id: $epochID) { referenceGasPrice } }",
+  "variables": {
+    "epochID": 123
+  }
+}'
+```
+
+### Working with variables
+
+Variables are a powerful way for passing different ObjectIDs, Digests, Sui Addresses, and other kind of data for full flexibility. Below is an example for querying the reference gas price for epoch 100, by passing the epoch as a variable:
+
+```graphql
+query ($epochID: Int!) {
+  epoch(id: $epochID) {
+    referenceGasPrice
+  }
+}
+variables {
+   "epochID": 100
+}
+
+And the result is:
+
+```json
+{
+  "data": {
+    "epoch": {
+      "referenceGasPrice": "780"
+    }
+  }
+}
+```
+
+:::info
+When working with the online GraphQL IDE, paste the variables' values in the `Variables` pane on the bottom left of the window, without the `variables` keyword.
+:::
```

Comment:
```suggestion

:::
```
Nit. I read somewhere that not having the empty line before closing and after opening`:::`s can cause issues in some corner cases.